app/assets/stylesheets/layouts/_dark_selectors.scss

```diff
@@ -143,11 +143,7 @@ table { border-color: #555555; } // TODO: what does this style
 
 /* - link-boxes */
 .link-box {
-  &.action-new { color: $font_color_linkbox_new; }
-  &.action-edit { color: $font_color_linkbox_edit; }
-  &.action-favorite { color: $font_color_linkbox_favorite; }
-  &.action-delete { color: $font_color_linkbox_delete; }
-  &.action-dismiss { color: $font_color_linkbox_dismiss; }
+  color: $font_color_linkbox;
```

Comment:
Does this have enough precedence?

Reply:
I think it would have shown up on the regression tests if it didn't, and either way the color doesn't actually need... differentiated